conf/application.conf

```diff
@@ -15,6 +15,16 @@
 # HOCON will fall back to substituting environment variable:
 #mykey = ${JAVA_HOME}
 
+## Digitalisiertedrucke
+
+index {
+	host = "localhost"
+	name = "digitalisiertedrucke"
+	location = "."
+	http_port = "6011"
```

Comment:
In organisations, we have them named like this: "index.es.port.http" -- with full stops. Do we have naming conventions for this? Do we care?